packages/expo-font/src/Font.ts

```diff
@@ -45,7 +45,7 @@ export function isLoaded(fontFamily: string): boolean {
  * Synchronously get all the fonts that have been loaded.
  * This includes fonts that were bundled at build time using the config plugin, as well as those loaded at runtime using `loadAsync`.
  *
- * @returns Returns array of font family names that have been loaded.
+ * @returns Returns array of strings which you can use as `fontFamily` [style prop](https://reactnative.dev/docs/text#style).
```

Comment:
I tried to document the returned value in greater details, such as 

> The values originate from several sources: font's file name (config plugin on Android), font's PostScript name (config plugin on iOS), and the name used to load the font with `loadAsync()`.

but it feels too clunky so ended up with this